Cargo.toml

```diff
@@ -205,11 +205,25 @@ members = [
 	"utils/frame/rpc/system",
 	"utils/prometheus",
 	"utils/wasm-builder",
-	# temp deps
-	"max-encoded-len",
-	"max-encoded-len/derive",
 ]
 
+[patch.crates-io.parity-scale-codec]
+git = "https://github.com/paritytech/parity-scale-codec"
+version = "2.2.0"
+branch = "igor-max-encoded-len"
+[patch.crates-io.parity-scale-codec-derive]
+git = "https://github.com/paritytech/parity-scale-codec"
+branch = "igor-max-encoded-len"
+version = "2.2.0"
+[patch.crates-io.primitive-types]
+git = "https://github.com/paritytech/parity-common"
+branch = "igor-prim-types-new-codec"
+version = "0.9.1"
+[patch.crates-io.impl-codec]
+git = "https://github.com/paritytech/parity-common"
+branch = "igor-prim-types-new-codec"
+version = "0.6.0"
```

Comment:
I suspect that these patches are no longer required.

Reply:
This still needs a patch to `primitive-types`, I'm on it.